CONTRIBUTING.md

```diff
@@ -12,11 +12,23 @@ contribution.
 
 - [Contributing Guidelines](#contributing-guidelines)
   * [Getting Started](#getting-started)
+    + [Consuming Development Versions of the Framework](#consuming-development-versions-of-the-framework)
   * [Tools](#tools)
   * [Workflows](#workflows)
     + [Adding code to support a new feature](#adding-code-to-support-a-new-feature)
+    + [Build and Validate Your Work](#build-and-validate-your-work)
+    + [Run Instrumentation Tests](#run-instrumentation-tests)
   * [Reporting Bugs/Feature Requests](#reporting-bugs-feature-requests)
   * [Contributing via Pull Requests](#contributing-via-pull-requests)
+  * [Troubleshooting](#troubleshooting)
+    + [Environment Debugging](#environment-debugging)
+    + [Problems with the Build](#problems-with-the-build)
+    + [Failing Instrumentation Tests](#failing-instrumentation-tests)
+  * [Related Repositories](#related-repositories)
```

Comment:
Idle thought: I'm surprised the CLI is not on this list given it is more pertinent to an Android application developer than the other libraries.

Reply:
Very true. Adding.